roles/openshift_health_checker/openshift_checks/mixins.py

```diff
@@ -0,0 +1,19 @@
+from openshift_checks import OpenShiftCheckException
+
+
+class NotContainerized(object):
```

Comment:
I worry a bit about mixins causing a bit of confusion with developers that aren't already, but I'm good with keeping it if it is the cleanest approach.

I think we'll need to think of enabling/disabling sets of classes more as a matrix, for example just considering atomic host/non-atomic host and containerized/non-containterized/system-containers:

|   | atomic | non-atomic |
|---|---|---|
| containerized |   |   |
| non-containerized  |   |
| system containers  |   |

I believe there will be similar matrices once we start tackling different options being enabled/disabled for each service type.